sentences/ca/_common.yaml

```diff
@@ -134,6 +134,11 @@ expansion_rules:
   posició: "{position}[ ][%|per[ ]cent]"
   percent: "[ ](%|per[ ]cent)"
   cancela: "cancel[·l]a[r]"
+  pausa: "(pausa[r]|para[r]|atura[r])"
```

Comment:
```suggestion
  pausa: "(pausa|para|atura)[r]"
```

Reply:
Oh, I didn't think about that one, but had to remove it since it was colliding with the order of stopping the TV.